types/src/on_chain_config/randomness_config.rs

```diff
@@ -41,6 +41,33 @@ impl AsMoveAny for ConfigV1 {
     const MOVE_TYPE_NAME: &'static str = "0x1::randomness_config::ConfigV1";
 }
 
+#[derive(Clone, Debug, Deserialize, PartialEq, Eq, Serialize)]
+pub struct ConfigV2 {
+    pub secrecy_threshold: FixedPoint64MoveStruct,
+    pub reconstruction_threshold: FixedPoint64MoveStruct,
+    pub fast_path_secrecy_threshold: FixedPoint64MoveStruct,
+}
+
+impl Default for ConfigV2 {
+    fn default() -> Self {
+        Self {
+            secrecy_threshold: FixedPoint64MoveStruct::from_u64f64(
+                U64F64::from_num(1) / U64F64::from_num(2),
+            ),
+            reconstruction_threshold: FixedPoint64MoveStruct::from_u64f64(
```

Comment:
what's the difference between secrecy and reconstruction?

Reply:
Any stakes < secrecy cannot reveal randomness. Any stakes > reconstruction can always reveal randomness. Due to rounding, there is a gap between these two.